### PR TITLE
[SAVANNAH SPRINT] Fix after first game, finish statistics

### PR DESCRIPTION
**Pull request by kejno**
Fix after first game, finish statistics

### CODE REVIEW

**Review by hallovarvara — CHANGES_REQUESTED** [review]
- [ ] Wrong word's quantity using in the game:
https://monosnap.com/file/Ouj20pMCwresFW5F3QU5O69ZDrSXlF
First time I chose to play with 5 words
After I tried 23, but all the time it's random (5, 9)

- [ ] If reload page, it will show 23, but play again for 5

Fix it please before merging.

File: rslang/src/basicComponents/Notification/NotificationView.jsx [hunk 1]
```diff
@@ -30,12 +30,12 @@ MyApp.propTypes = {
...
 export default function IntegrationNotistack(props) {
   const classes = {
-    root: 'notification-root',
+    root: `notification-root notification_${props.variant}`,
   };
 
   const {
     afterClose = () => {},
-    duration = 5000,
+    duration = 50000,
```
Comment (DmitryLebetsky) [inline rslang/src/basicComponents/Notification/NotificationView.jsx]: 50s?

File: rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss [hunk 2]
```diff
@@ -1,32 +1,32 @@
...
+.Title{
+  margin: 0 0 10px 43px;
+  color: rgb(82, 82, 82);
+}
+
+.Content{
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  padding: 4px 0;
```
Comment (DmitryLebetsky) [inline rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss]: You can use our mixins @include flex(column, flex-start, flex-start)

File: rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
-  
\ No newline at end of file
+}
+.mistakeTitle{
+  color: gray;
+}
+.Total{
+  padding: 0 10px;
+  margin-left: 10px;
+  color: #ffffff;
```
Comment (DmitryLebetsky) [inline rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss]: $color-light

File: rslang/src/components/Sprint/Sprint.jsx [hunk 4]
```diff
@@ -193,12 +193,14 @@ class Sprint extends Component {
...
       clearTimeout(timerId);
-      this.setState({ isFinished: true });
+      this.setState({ isFinished: true, });
       saveSessionInfoToLocal(applicationThings.SPRINT);
+
     }
   }
 
   onClickHandler = (e) => {
+    console.log(initialState)
```
Comment (DmitryLebetsky) [inline rslang/src/components/Sprint/Sprint.jsx]: Don't forget to remove

File: rslang/src/scss/components/_notification.scss [hunk 5]
```diff
@@ -1,21 +1,27 @@
...
+    }
   }
 
-  .SnackbarItem-variantError-21 {
-    background-color: $color-error;
+  &.notification_error {
+    .MuiSnackbarContent-root,
+    .MuiPaper-root {
+      background-color: $color-error;
+    }
```
Comment (DmitryLebetsky) [inline rslang/src/scss/components/_notification.scss:18]: Use prefix like -savannah to restyle notifications, please

**Review by DmitryLebetsky — APPROVED** [review]

File: rslang/src/basicComponents/Notification/NotificationView.jsx (hunk 1 above)
Comment (hallovarvara) [reply rslang/src/basicComponents/Notification/NotificationView.jsx]: it's mine, fixed.

File: rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss (hunk 2 above)
Comment (hallovarvara) [reply rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss]: fixed

File: rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss (hunk 3 above)
Comment (hallovarvara) [reply rslang/src/components/Savannah/components/FinishPage/ResultList/ResultList.module.scss]: fixed

File: rslang/src/components/Sprint/Sprint.jsx (hunk 4 above)
Comment (hallovarvara) [reply rslang/src/components/Sprint/Sprint.jsx]: fixed

File: rslang/src/scss/components/_notification.scss (hunk 5 above)
Comment (hallovarvara) [reply rslang/src/scss/components/_notification.scss:18]: It's mine, correct it here just for rapidly finishing project. We need it in the app.

**Pull request merged**